docs/src/pages/[platform]/connected-components/account-settings/delete-user/react.mdx

```diff
@@ -1,2 +1,273 @@
-## Delete User
-TODO
+import { Alert, Expander, ExpanderItem, ScrollView } from '@aws-amplify/ui-react';
+
+import ReactPropsTable from '@/components/propsTable/ReactPropsTable';
+import { Example, ExampleCode } from '@/components/Example';
+import { DELETE_USER, OVERRIDES, WARNING, DELETE_BUTTON } from './props';
+import { ERROR_MESSAGE } from '../props';
+
+### Props
+
+<ReactPropsTable props={DELETE_USER} />
+
+### Basic Usage
+
+`DeleteUser` has `onSuccess` handler that will be called after successful user account deletion. You may use this callback to run any custom handling, such as navigating route or rendering confirmation messages.
+
+<Example>
+  <ExampleCode>
+
+```jsx
+import React from 'react';
+import { AccountSettings } from '@aws-amplify/ui-react';
+
+function App() {
+  const handleSuccess = () => {
+    alert('password is successfully changed!')
+  }
+  
+  return (
+    <AccountSettings.DeleteUser onSuccess={handleSuccess} />
+  );
+}
+```
+
+  </ExampleCode>
+</Example>
+
+### Override default delete handler
+You can override the default deleteUser handler with `handleDelete` prop to add any custom cleanup handlers before or after account deletion.
```

Comment:
nit: "`deleteUser` handler"

Reply:
```suggestion
You can override the default `deleteUser` handler with `handleDelete` prop to add any custom cleanup handlers before or after account deletion.
```

Reply:
`radii` for `medium` changed in 65388d0a1